recipes-kernel/linux/linux-legacy-arm-dts.inc

```diff
@@ -0,0 +1,7 @@
+# In linux kernels < 6.6 dts files for arm arch are collected in the
+# common dts directory arch/arm/boot/dts
+
+KERNEL_DEVICETREE:imx23evk = "imx23-evk.dtb"
+KERNEL_DEVICETREE:imx28evk = "imx28-evk.dtb"
+KERNEL_DEVICETREE:imx51evk = "imx51-babbage.dtb"
+KERNEL_DEVICETREE:imx53qsb = "imx53-qsb.dtb imx53-qsrb.dtb"
```

Comment:
Using a Python function to change variables is generic, so we avoid adding each variable individually.

Reply:
I've got a function that I've been working on that I can contribute.

Reply:
See #1685